pyignite/connection/aio_connection.py

```diff
@@ -158,7 +160,7 @@ async def _connect_version(self) -> Union[dict, OrderedDict]:
             await self._send(stream.getbuffer(), reconnect=False)
 
         with AioBinaryStream(self.client, await self._recv(reconnect=False)) as stream:
-            hs_response = await HandshakeResponse.parse_async(stream, self.protocol_version)
+            hs_response = await HandshakeResponse.parse_async(stream, self.protocol_context)
```

Comment:
We should update from response `protocol_context.features`

Reply:
Good catch. Fixed.